src/irmin-unix/info.mli

```diff
@@ -14,5 +14,8 @@
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  *)
 
-val v :
-  ?author:string -> ('a, Format.formatter, unit, Irmin.Info.f) format4 -> 'a
+module Make (S : Irmin.S) : sig
+  include Irmin.Info.S with type t = S.info
```

Comment:
```suggestion
module Make (Info : Irmin.Info.S) : sig
  include Info.S with type t = Info.t
```

Looks like this could be a functor over Infos.

Reply:
Fixed